dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,582 @@
+several parts. Those parts can include:
+
+----
+<registry_host>/<user_name>/<image_name>:<tag>
+----
+
+The `<user_name>` part in the above could also refer to a
+link:../architecture/core_concepts/projects_and_users.html#projects[project] or
+link:../architecture/core_concepts/projects_and_users.html#namespaces[namespace]
+if the image is being stored in an {product-title} or Kubernetes environment.
+
+{product-title} provides the `oc tag` command, which is similar to the `docker
+tag` command, but operates on image streams instead of directly on images.
+
+[NOTE]
+====
+See Red Hat Enterprise Linux 7's
+https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/version-7/getting-started-with-containers/#creating_docker_images[Getting
+Started with Containers] documentation for more about tagging images directly
+using the `docker` CLI.
+====
+
+[[adding-tag]]
+=== Adding Tags to Image Streams
+
+Keeping in mind that an image stream in {product-title} comprises one or more
+Docker images identified by tags, an image stream can also be configured so that
+a tag "tracks" another one.
+
+For example, to configure the *ruby* image's *latest* tag to always refer to the
+current image for the tag *2.0*:
+
+====
+----
+$ oc tag ruby:latest ruby:2.0
+----
+====
+
+[[tag-removal]]
+=== Removing Tags from Image Streams
+
+You can also stop tracking a tag by removing the tag. For example, to stop
+tracking an existing *latest* tag:
+
+====
+----
+$ oc tag -d ruby:latest
+----
+====
+
+However, while the above command removes the tag from the image stream
+definition, it does not remove it from the image stream status. The image stream
+definition is user-defined, whereas the image stream status reflects the
+information the system has from the specification.
+
+To remove a tag completely from an image stream:
+
+====
+----
+$ oc delete istag/ruby:latest
+----
+====
+
+[[referencing-images-in-image-streams]]
+== Referencing Images in Image Streams
+
+When defining tags in an image stream, you can decide if the tag should be
+imported or just referenced from an image stream. This is achieved by setting
+`*reference*`. By default `*reference*` is *false*, which means the tag's
+metadata will be imported during the import. Setting it to *true* on the other
+hand, will mark the tag as a reference.
+
+The following references are available:
+
+- An `*ImageStreamTag*` is used to reference or retrieve an image for a given
+image stream and tag. It uses the following convention for its name:
++
+----
+<image_stream_name>:<tag>
+----
+
+- An `*ImageStreamImage*` is used to reference or retrieve an image for a given
+image stream and image name. It uses the following convention for its name:
++
+----
+<image_stream_name>@<name>
+----
+
+- A `*DockerImage*` is used to reference or retrieve an image for a given external
+registry. It uses standard Docker _pull specification_ for its name, e.g.:
++
+----
+openshift/ruby-20-centos7:2.0
```

Comment:
Re: "third-party", I guess I'm asking because the Red Hat Registry wouldn't have struck me as third-party; it is external to the cluster, but we mention "external registry" in the DockerImage explanation above.

Also BTW, should that "DockerImage" above be "dockerImageReference"?